pylint/checkers/unsupported_version.py

```diff
@@ -79,6 +93,51 @@ def _check_typing_final(self, node: nodes.Decorators) -> None:
                 "using-final-decorator-in-unsupported-version", node=decorator
             )
 
+    @only_required_for_messages("using-exception-groups-in-unsupported-version")
+    def visit_trystar(self, node: nodes.TryStar) -> None:
+        if not self._py311_plus:
+            self.add_message("using-exception-groups-in-unsupported-version", node=node)
```

Comment:
I'm not sure we can even visit a trystar if we're not in python 3.11+ (the ast for 3.10 does not have the construct, we had to add it at this point, right ?)

Reply:
Right. But we need to know whether the lowest supported version is 3.11 or lower.